pytrip/vdx.py

```diff
@@ -1279,7 +1279,9 @@ def read_vdx_old(self, slice_number, xy_line):
         xy_pairs = [xy_line[i:i + 2] for i in range(0, len(xy_line), 2)]  # make list of pairs
         for x, y in xy_pairs:
             # TRiP98 saves X,Y coordinates as integers, to get [mm] they needs to be divided by 16
-            self.contour.append([float(x) / 16.0, float(y) / 16.0, float(slice_number)])
+            self.contour.append([self.cube.pixel_size * float(x) / 16.0,
```

Comment:
Reading vdx without dicom is still supported in many places, but not here.
There should be an if switch to deal with situation when self.cube is None